node/os.ts

```diff
@@ -0,0 +1,11 @@
+import os from "os";
+
+export function verifyOS(): void {
+  const isMac = os.platform() === "darwin";
+  if (!isMac)
+    throw new Error(
+      "Verification must be performed on macos! The easiest way to do this is with a github action. See https://github.com/CycleOperators/BalanceCheckerVerification for github action instructions. This is due to differences in build environments resulting in different wasm hashes."
```

Comment:
```suggestion
      "To verify the controllers and hash of the blackholed CycleOps Balance Checker canister from a non-macOS operating system, use the following GitHub action. See https://github.com/CycleOperators/BalanceCheckerVerification for github action instructions.
      
      Currently, local verification is only supported on macOS! This is due to differences in build environments resulting in different wasm hashes.     
```

Reply:
Call to action first with link for easier DevX (who reads error messages these days 😅 )

Reply:
This seems like a longer way to say the same thing to me.